applications/Mesh light/mesh_light.json

```diff
@@ -35,6 +35,36 @@
           ]
         }
       ]
+    },
+    {
+      "version": "2.3.0",
+      "release_notes": "Initial release",
```

Comment:
replace initial release with "- Enabled the CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE Kconfig option."

Reply:
And "\n- Added support for running the light switch as a Low Power node."